recipes/dsfmt/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+make std
+make std-check
+make sse2-check
+
+gcc -O3 -finline-functions -fomit-frame-pointer -DNDEBUG -DDSFMT_MEXP=19937 \
+  -fPIC -fno-strict-aliasing --param max-inline-insns-single=1800 -Wmissing-prototypes \
+  -Wall -std=c99 -shared dSFMT.c -o libdSFMT.${SHLIB_EXT}
+
+mkdir ${PREFIX}/lib
+mkdir ${PREFIX}/include
```

Comment:
Normally I do `mkdir -p` just in case it is already there. Not that it would be in this case. We just don't want to waste time debugging something we are doing just in case these directories are not there.